### ISSUE TITLE
Unable to stream game(s?).

### ISSUE THREAD

**Issue by rokejulianlockhart** [ValveSoftware/steam-for-linux#9116]
#### Your system information

* Steam client version (build number or date): 2022-12-15T21:27:25
* Distribution (e.g. Ubuntu): Fedora Workstation 6.1.6-100.fc36.x86_64
* Opted into Steam client beta?: [Yes/No]
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
```
Error Reference: Library_7728431_88af7cf66459d659
Cannot read property 'TakeFocus' of undefined
```
[appears](https://user-images.githubusercontent.com/42837531/212964317-be1962ff-07f3-40ac-9909-ab19eb0f4b8b.png) when I attmpt to stream [OBS Studio](http://store.steampowered.com/app/1905180).

#### Steps for reproducing this issue:

1. Install [OBS Studio](http://store.steampowered.com/app/1905180) onto the most recent Windows 11 Insider Preview build.
2. Attempt to stream it from the aforementioned system.

**Comment by schematical** [comment]
Same problem. Just got it tonight but this is the first time I tried in a while.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/ValveSoftware/steam-for-linux/issues/9116#issuecomment-1401991290

Yeah, I'm not, @noobmagnet. I hadn't been able to locate the button, since I was convinced that it wouldn't be in such a prominent location:

> ![image](https://user-images.githubusercontent.com/42837531/214377846-b4b99bd3-e591-4d59-8766-c82d8e81bd06.png)
